.github/workflows/tests.yml

```diff
@@ -126,10 +132,338 @@ jobs:
           push: true
           tags: ${{ needs.tag.outputs.image }}
 
+  test-uv-install:
+    name: Test uv installs (CPU)
+    runs-on: ubuntu-latest
+    needs: [lint]
+    strategy:
+      fail-fast: false
+      matrix:
+        extras: ['', 'determined', 'api', 'openai', 'transformers', 'accelerate', 'comet', 'optional', 'all']
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+
+      - name: Install uv
+        run: |
+          curl -LsSf https://astral.sh/uv/install.sh | sh
+
+      - name: Install package via
+        run: |
+          if [ "${{ matrix.extras }}" != "" ]; then
+            uv sync --extra ${{ matrix.extras }}
```

Comment:
I think we can skip the `uv`-specific install tests: Since we already have a lock file that means we know that `uv` can resolve all dependencies successfully. Given that, I don't see how these tests could fail unless someone renames extras.